### PR TITLE
Fix slot_name

### PR DESCRIPTION
**Pull request by darcy-shen**

### CODE REVIEW

File: src/Edit/Interface/edit_interface.hpp [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 #ifndef EDIT_INTERFACE_H
 #define EDIT_INTERFACE_H
+
```
Comment (darcy-shen) [inline src/Edit/Interface/edit_interface.hpp:14]: just want to make it better

File: src/Graphics/Gui/widget.cpp [hunk 2]
```diff
@@ -39,6 +39,7 @@ slot_name (const slot s) {
     "SLOT_MOUSE_POINTER",
     "SLOT_INVALIDATE",
     "SLOT_INVALIDATE_ALL",
+    "SLOT_INVALID",
```
Comment (darcy-shen) [inline src/Graphics/Gui/widget.cpp:42]: https://github.com/XmacsLabs/mogan/blob/6c6bf6be2fe912de44c78c2ed189255df87cd705/src/Graphics/Gui/message.hpp#L43:L42

Comment (PikachuHy) [reply src/Graphics/Gui/widget.cpp:42]: how `SLOT_INVALID` work?

Comment (darcy-shen) [reply src/Graphics/Gui/widget.cpp:42]: I know about the slots, but I have not read the code about every slot.

It is a design of the server/client arch.

**Review by PikachuHy — APPROVED** [review]
LGTM

**Pull request merged**